jujugui/static/gui/src/app/user/user.js

```diff
@@ -26,24 +26,35 @@ if (typeof this.jujugui === 'undefined') {
 const User = class User {
 
   constructor(cfg = {}) {
-    this.storage = cfg.storage || sessionStorage;
+    // We pass in these values to make test setup easier.
+    this.sessionStorage = cfg.sessionStorage || sessionStorage;
+    this.localStorage = cfg.localStorage || localStorage;
+    this._DISCHARGE_TOKEN = 'discharge-token';
```

Comment:
Is this required to be a class attribute or can it just be a const defined in the file but unreachable from outside?